app_dart/lib/src/service/luci.dart

```diff
@@ -168,4 +229,7 @@ class LuciTask {
 
   /// The status of this task. See the [Task] class for supported values.
   final String status;
+
+  /// The build number of this task.
+  final int buildNumber;
```

Comment:
It's more readable to add a `LuciBuilder` field since every LuciTask must have been executed by a builder.

And add a method to return the branch calculated from `ref`.

Reply:
Please see the comment above.